.github/workflows/merge-translations.yml

```
@@ -0,0 +1,87 @@
+name: Merge translations
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+permissions: {}
+
+env:
+  WEBLATE_HOST: 'https://hosted.weblate.org'
+  WEBLATE_COMPONENT: 'immich/immich'
+
+jobs:
+  merge:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
```

I think we don't need this since we're using the push_o_matic token?

```suggestion
      pull-requests: read
```